docs: documentation overhaul (#1651)
GIScience/openrouteservice: docs/run-instance/configuration/preparation.md

> # Preparation settings

Reviewer: This doesn't show up in the sidebar

| key                      | type   | description                                                       | default value |
|--------------------------|--------|-------------------------------------------------------------------|---------------|
| min_network_size         | number | Minimum size (number of edges) of an independent routing subgraph | `200`         |

> | min_one_way_network_size | number |                                                                   | `200`         |

Reviewer: Description missing

Reviewer 2: Removed as unsupported since GH1.0

> | methods                  | object | [methods properties](#preparationmethods)                         |               |

Reviewer: ```suggestion
| methods                  | object | [methods properties](#preparation-methods)                         |               |
```

## **preparation.methods**

| key  | type   | description                                            | example value                   |
|------|--------|--------------------------------------------------------|---------------------------------|

> | ch   | object | Settings for preprocessing contraction hierarchies     | [ch](#preparationmethodsch)     |

Reviewer: ```suggestion
| ch   | object | Settings for preprocessing contraction hierarchies     | [ch](#preparation-methods-ch)     |
```

> | lm   | object | Settings for preprocessing A* with landmarks           | [lm](#preparationmethodslm)     |

Reviewer: ```suggestion
| lm   | object | Settings for preprocessing A* with landmarks           | [lm](#preparation-methods-lm)     |
```

> | core | object | Settings for preprocessing core routing with landmarks | [core](#preparationmethodscore) |

Reviewer: ```suggestion
| core | object | Settings for preprocessing core routing with landmarks | [core](#preparation-methods-core) |
```

## **preparation.methods.ch**

| key        | type    | description                                              | example value           |
|------------|---------|----------------------------------------------------------|-------------------------|
| enabled    | boolean | Enables or disables the routing algorithm                | `true`                  |
| threads    | number  | Number of parallel threads for computing the preparation | `1`                     |

> | weightings | string  | Comma-separated list of weightings                       | `recommended,shortests` |

Reviewer: ```suggestion
| weightings | string  | Comma-separated list of weightings                       | `recommended,shortest` |
```

## **preparation.methods.core**

| key        | type    | description                                                                                                               | example value                                               |
|------------|---------|---------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
| enabled    | boolean | Enables or disables the routing algorithm                                                                                 | `true`                                                      |
| threads    | number  | Number of parallel threads for computing the preparation                                                                  | `1`                                                         |
| weightings | string  | Comma-separated list of weightings                                                                                        | `recommended,shortest`                                      |

> | landmarks  | number  | Total number of precomputed landmarks, the subset used during the query is set in `execution.methods.lm.active_landmarks` | `32`                                                        |

Reviewer: ```suggestion
| landmarks  | number  | Total number of precomputed landmarks, the subset used during the query is set in `execution.methods.core.active_landmarks` | `32`                                                        |
```
… I am confused :D
